mcaleb808/myPharmBackend
# Exit the Server Instance After running Tests

---

#### Description of changes made

> This implementation fixes the issue with the server that was kept open even after the tests were run.
> We should shut it down after all tests are run.

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/mcaleb808/myPharmBackend.git), and `cd into the project`
- Check out this branch by _git checkout `exit-the-server-after-testing`_
- Run `yarn | npm install` to install the project dependencies
- Make sure to have your `.env` file set up correctly
- Run `yarn dev | npm run dev` to start the local server.

#### Checklist:


#### GitHub issue your changes fix

- None

#### Screenshots

|        BEFORE        |       AFTER       |
| :------------------: | :---------------: |
| <img width="862" alt="Screen Shot 2019-12-12 at 12 19 01" src="https://user-images.githubusercontent.com/29999424/70703888-a29e8480-1cd9-11ea-9da5-0af11d45d051.png">|<img width="788" alt="Screen Shot 2019-12-12 at 12 17 11" src="https://user-images.githubusercontent.com/29999424/70703895-a92cfc00-1cd9-11ea-9313-187e54a0f3be.png">|
